### PR TITLE
Endpoint fixes

### PR DESCRIPTION
**Pull request by iOliver678**

### CODE REVIEW

**Review by pnavab — APPROVED** [review]
looks good

File: sqlite_helpers.py [hunk 1]
```diff
@@ -37,14 +32,19 @@ def deleteUrl(alias):
...
+    for row in allItems:
+        holder = {"id": row[0], "url": row[1], "alias": row[2], "timestamp": row[3]}
+        listData.append(holder)
+    return listData
     
 
 def findUrl(alias):
     query = f"SELECT url FROM urls WHERE alias='{alias}'"
     res = cursor.execute(query)
     item = res.fetchone()
```
Comment (pnavab) [inline sqlite_helpers.py]:
```suggestion
    item = res.fetchone()[0]
```

File: sqlite_helpers.py [hunk 2]
```diff
@@ -37,14 +32,19 @@ def deleteUrl(alias):
...
+    item = item[0]
```
Comment (pnavab) [inline sqlite_helpers.py]:
```suggestion
```

**Comment by iOliver678** [comment]
> looks good

Fixed code suggestions

**Pull request merged**